### PR TITLE
feat(Header/poller): display command in poller top counter menu first part

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
<details><summary>Description</summary>
In Centreon Cloud, users create their pollers by executing a shell command that we provide him by email.

It would be very convenient that this command be displayed somewhere in the UI of the platform.
I think that the poller listing could be a good candidate.

Salt could easily write the command in a file during central setup.
The backend would have to read it to display it in the UI.
https://centreon.atlassian.net/jira/software/c/projects/MON/boards/211?modal=detail&selectedIssue=MON-12467&assignee=62beb9132b6e4ddba115721f
</details>

**PR closed**  : change on the bacend side, api will be implemented at cloud extension method (pr will be migrated to cloud extension)

**screens**
check snapshots

**video**

[installCommand.webm](https://github.com/centreon/centreon/assets/97687698/a68ee27a-b9df-4e9f-9cd4-bceb99dd2e36)

### CODE REVIEW

File: centreon/www/front_src/src/Header/Poller/PollerSubMenu/InstallCommand/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  const { data, error } = useFetchQuery({
+    decoder: installCommandDecoder,
+    getEndpoint: () => installCommandEndpoint,
+    getQueryKey: () => ['installCommandPollers']
+  });
+
+  const installCommandRef = useRef<HTMLTextAreaElement>();
+
+  const onCopy = (): void => {
```
Comment (Thebarda) [inline centreon/www/front_src/src/Header/Poller/PollerSubMenu/InstallCommand/index.tsx:33]:
```suggestion
  const copy = (): void => {
```

File: centreon/www/front_src/src/Header/Poller/PollerSubMenu/InstallCommand/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    );
+  };
+
+  if (!data || error) {
+    return null;
+  }
+
+  return (
+    <>
+      <ListItemButton onClick={onCopy}>
```
Comment (Thebarda) [inline centreon/www/front_src/src/Header/Poller/PollerSubMenu/InstallCommand/index.tsx:55]:
```suggestion
      <ListItemButton onClick={copy}>
```

File: centreon/www/front_src/src/Header/Poller/PollerSubMenu/InstallCommand/index.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+  const installCommandRef = useRef<HTMLTextAreaElement>();
+
+  const onCopy = (): void => {
+    if (!installCommandRef || !installCommandRef?.current) {
+      return;
+    }
+
+    const value = installCommandRef.current?.value;
+    navigator.clipboard.writeText(value).then(
```
Comment (Thebarda) [inline centreon/www/front_src/src/Header/Poller/PollerSubMenu/InstallCommand/index.tsx:39]: Use the `useCopyToClipboard` hook from Centreon UI

File: centreon/www/front_src/src/Header/Poller/translatedLabels.ts [hunk 4]
```diff
@@ -28,3 +28,8 @@ export const labelPollerRunning = 'Everything is OK';
 export const labelConfigurePollers = 'Configure pollers';
 export const labelAllPollers = 'All pollers:';
 export const labelPollers = 'Pollers';
+export const labelInstallCommand = 'Install command';
```
Comment (bdauria) [inline centreon/www/front_src/src/Header/Poller/translatedLabels.ts:31]:
```suggestion
export const labelInstallCommand = 'Copy install command';
```

**Pull request closed without merging**